_pytest/mark.py

```diff
@@ -335,3 +339,6 @@ def add_mark(self, mark):
     def __iter__(self):
         """ yield MarkInfo objects each relating to a marking-call. """
         return imap(MarkInfo, self._marks)
+
+
+MARK_GEN = MarkGenerator()
```

Comment:
i#d love a better name for that object in general, nothing good comes to mind tho